src/Umbraco.Tests.Integration/Logging/LogviewerTests.cs

```diff
@@ -34,10 +35,11 @@ public void Setup()
         {
             //Create an example JSON log file to check results
             //As a one time setup for all tets in this class/fixture
-            var ioHelper = TestHelper.IOHelper;
-            var hostingEnv = TestHelper.GetHostingEnvironment();
+            var helper = new TestHelper();
```

Comment:
Would probably name the var `testHelper` as we have many helpers 🙂

Reply:
I've renamed it to testHelper, thanks! :D